samples/peptide_mixture_grxmc_ideal.py

```diff
@@ -106,7 +105,7 @@
 # Solution parameters
 c_salt=5e-3 * pmb.units.mol/ pmb.units.L
 
-if args.mode == 'standard':
+if args.mode == 'standard' or args.mode == 'standard-test':
```

Comment:
and lines 119,142,147,166, 168. I think that you can drop the  test-related checks because it only matters if you are running the test for the last lines of the script